PCU/src/main/java/com/coffeelovers/pcu/model/Product.java

```diff
@@ -0,0 +1,86 @@
+package com.coffeelovers.pcu.model;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.Table;
+
+@Entity
+@Table(name="products")
```

Comment:
the name of the table should be without "s"
```suggestion
@Table(name="product")
```